tidb-cloud/serverless-faqs.md

```diff
@@ -53,12 +53,20 @@ Usage beyond the free quota will be charged. Once the free quota of a cluster is
 
 For more information, see [TiDB Serverless usage quota](/tidb-cloud/select-cluster-tier.md#usage-quota).
 
+### What are the limitations of the free plan?
+
+The performance of a cluster under the free plan is capped at a certain level, allowing for a maximum of 10,000 RUs per second based on the actual workload. It is important to note that memory allocation per query is also limited to 256MiB for the free plan. To maximize cluster performance, you can choose to enable the commercial offering by [increasing the spending limit](/tidb-cloud/manage-serverless-spend-limit.md#update-spending-limit).
+
 ### How can I estimate the number of RUs required by my workloads and plan my monthly budget?
 
-To get the RU consumption of individual SQL statements, you can use the [`EXPLAIN ANALYZE`](/sql-statements/sql-statement-explain-analyze.md) SQL statement. 
+To get the RU consumption of individual SQL statements, you can use the [`EXPLAIN ANALYZE`](/sql-statements/sql-statement-explain-analyze.md) SQL statement. However, it is important to note that the RUs usage returned in EXPLAIN ANALYZE does not incorporate egress RUs, as egress usage is measured separately in the gateway, which is unknown to the TiDB server.
 
 To get the RUs and storage used by your cluster, view the **Usage this month** pane on your cluster overview page. With your past resource usage data and real-time resource usage in this pane, you can track your cluster's resource consumption and estimate a reasonable spending limit. If the free quota cannot meet your requirement, you can edit the spending limit easily. For more information, see [Manage Spending Limit for TiDB Serverless clusters](/tidb-cloud/manage-serverless-spend-limit.md).
 
+### How can I optimize my workload to minimize the number of RUs consumed?
+
+Ensure that your queries have been carefully optimized for optimal performance by following the [Optimizing SQL Performance](/develop/dev-guide-optimize-sql-overview.md) guidelines. In addition, minimizing the amount of egress traffic is also crucial for reducing RUs consumption. To achieve this, it is recommended to return only the required columns and rows in your query, which in turn helps reduce network egress traffic. This can be achieved by carefully selecting and filtering the columns and rows to be returned, thereby optimizing network utilization.
```

Comment:
```suggestion
Ensure that your queries have been carefully optimized for optimal performance by following the guidelines in [Optimizing SQL Performance](/develop/dev-guide-optimize-sql-overview.md). In addition, minimizing the amount of egress traffic is also crucial for reducing RUs consumption. To achieve this, it is recommended to return only the necessary columns and rows in your query, which in turn helps reduce network egress traffic. This can be achieved by carefully selecting and filtering the columns and rows to be returned, thereby optimizing network utilization.
```